src/docs/implementation/testing.md

```diff
@@ -26,7 +26,9 @@ namespace Tests
         [Fact]
         public async Task SaysHelloCorrectly()
         {
-            var cluster = new TestCluster();
+            var builder = new TestClusterBuilder();
+            builder.Options.ServiceId = Guid.NewGuid().ToString();
```

Comment:
This property should not need to be set manually - it defaults to `Guid.NewGuid().ToString("N")`